spec/features/social_media_meta_tags_spec.rb

```diff
@@ -37,19 +34,43 @@
 
       visit root_path
 
-      expect(page).to have_css 'meta[name="keywords"][content="' + meta_keywords + '"]', visible: false
-      expect(page).to have_css 'meta[name="twitter:site"][content="' + twitter_handle + '"]', visible: false
-      expect(page).to have_css 'meta[name="twitter:title"][content="' + meta_title + '"]', visible: false
-      expect(page).to have_css 'meta[name="twitter:description"][content="' + sanitized_truncated_meta_description + '"]', visible: false
-      expect(page).to have_css 'meta[name="twitter:image"][content="http://www.example.com/social_media_cambia_tu_ciudad_twitter.jpg"]', visible: false
-      expect(page).to have_css 'meta[property="og:title"][content="' + meta_title + '"]', visible: false
-      expect(page).to have_css 'meta[property="article:publisher"][content="' + url + '"]', visible: false
-      expect(page).to have_css 'meta[property="article:author"][content="https://www.facebook.com/' + facebook_handle + '"]', visible: false
-      expect(page).to have_css 'meta[property="og:url"][content="http://www.example.com/"]', visible: false
-      expect(page).to have_css 'meta[property="og:image"][content="http://www.example.com/social_media_cambia_tu_ciudad.jpg"]', visible: false
-      expect(page).to have_css 'meta[property="og:site_name"][content="' + org_name + '"]', visible: false
-      expect(page).to have_css 'meta[property="og:description"][content="' + sanitized_truncated_meta_description + '"]', visible: false
+      expect(page).to have_css 'meta[name="keywords"][content="' + meta_keywords + '"]',
+                                visible: false
```

Comment:
Another option would be to add a matcher in `spec/support/matchers/have_meta.rb`:

```ruby
RSpec::Matchers.define :have_meta do |name, content|
  match do
     has_css?("meta[name='#{name}'][content='#{content}']", visible: false)
  end

  failure_message do
    meta = first("meta[name='#{name}']", visible: false)

    if meta
      "expected to find meta tag #{name} with '#{content}', but had '#{meta[:content]}'"
    else
      "expected to find meta tag #{name} but there were no matches."
    end
  end
end
```
And then these lines would read like:
```ruby
      expect(page).to have_meta :keywords, meta_keywords
```